### PR TITLE
chore: fix markdown linting

### PR DESCRIPTION
**Pull request by AlekSi**
* without `-print0` / `-0`, paths with spaces were skipped sometimes;
* `--stdin-filename` flag incorrectly consumed the first file.

### CODE REVIEW

File: .textlintrc.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "rules": {
+    "one-sentence-per-line": true
+  },
+  "filters": {
+    "comments": true
```
Comment (AlekSi) [inline .textlintrc.json:6]: This cannot be specified via flags

File: Dockerfile [hunk 2]
```diff
@@ -624,7 +626,8 @@ RUN find . \
     -not -path '*/node_modules/*' \
     -not -path './hack/chglog/**' \
     -not -path './website/content/docs/*/Reference/*' \
-    | xargs textlint --rule one-sentence-per-line --stdin-filename
```
Comment (AlekSi) [inline Dockerfile:627]: `--stdin-filename` incorrectly consumed the first file

File: website/content/docs/v0.10/Bare Metal Platforms/equinix-metal.md [hunk 3]
```diff
@@ -27,10 +27,14 @@ The following is a list of kernel parameters required by Talos:
 
 ### User Data
 
+<!-- textlint-disable one-sentence-per-line -->
+
 To configure a Talos you can use the metadata service provide by Equinix Metal.
 It is required to add a shebang to the top of the configuration file.
 The shebang is arbitrary in the case of Talos, and the convention we use is `#!talos`.
```
Comment (AlekSi) [inline website/content/docs/v0.10/Bare Metal Platforms/equinix-metal.md:34]: https://github.com/IQTLabs/textlint-rule-one-sentence-per-line/issues/7

**Review by smira — APPROVED** [review]

**Pull request merged**